src/containers/AccountConnection.jsx

```diff
@@ -266,7 +285,7 @@ class AccountConnection extends Component {
                   />
                 )}
               </ul>}
-            {!connector.dataType &&
+            {!(connector.dataType && connector.dataType.length) &&
```

Comment:
Alternative:
```js
const hasDataTypes = connector.dataType && connector.dataType.length
// [...]
{!hasDataTypes
```
I personnaly read only the left part of affectations. The right part is only for debug or evolutions.